src/napari_workflows/_undo_redo_functionality.py

```diff
@@ -5,7 +5,7 @@
 from typing import List, Callable
 
 import warnings
-from ._workflow import Workflow, _layer_name_or_value
+from ._workflow import Workflow
```

Comment:
```suggestion
from ._workflow import Workflow, _layer_name_or_value
```
Please do not do any backwards-compatibility breaking changes; in particular if they are unrelated to the pull-request title / description.

Reply:
Good point. Reverted.